deployment_scripts/deploy_windows.ps1

```diff
@@ -162,29 +149,10 @@ function Deploy-Windows([String] $monkey_home = (Get-Item -Path ".\").FullName,
     & npm run dist
     Pop-Location
 
-    # Install pywin32
-    "Downloading pywin32"
-    $webClient.DownloadFile($PYWIN32_URL, $TEMP_PYWIN32_INSTALLER)
-    Start-Process -Wait $TEMP_PYWIN32_INSTALLER -ErrorAction Stop
-    Remove-Item $TEMP_PYWIN32_INSTALLER
-
     # Create infection_monkey/bin directory if not already present
     $binDir = (Join-Path -Path $monkey_home -ChildPath $MONKEY_DIR | Join-Path -ChildPath "\bin")
     New-Item -ItemType directory -path $binaries -ErrorAction SilentlyContinue
 
-    # Download upx
-    if(!(Test-Path -Path (Join-Path -Path $binDir -ChildPath "upx.exe") )){
```

Comment:
Why are you removing it?

Reply:
I couldn't make `--upx-exclude` work. Since we are not using upx, we might as well don't download it.